File: code/sample.m
// Objective-C Sample
#import <Foundation/Foundation.h>

@interface Greeter : NSObject
- (NSString *)greet:(NSString *)name;
@end

@implementation Greeter
- (NSString *)greet:(NSString *)name {
    return [NSString stringWithFormat:@"Hello, %@!", name];
}
@end

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        Greeter *g = [[Greeter alloc] init];
        NSLog(@"%@", [g greet:@"World"]);
    }
    return 0;
}

File: code/sample.jl
# Julia Sample
function greet(name::String)
    println("Hello, $name!")
end

function fibonacci(n::Int)
    n <= 1 && return n
    return fibonacci(n - 1) + fibonacci(n - 2)
end

greet("World")
println("Fibonacci: ", [fibonacci(i) for i in 0:9])
